### DISCUSSION TITLE
Setup with npm

### DISCUSSION THREAD

**Discussion by Ristophonics** [fancyapps/ui#53] [Q&A] [answered]
Hi All,
I am having a hard time getting V4 running. Using only HTML/CSS/JavaScript. I can see the package in my node_modules and I am using the correct import statements but I get an error when trying to open the page using Express. Any other noobs having similar difficulty? I know there is an easy solution and I just don't know what I am doing wrong. Any ideas?

**Comment by fancyapps** [comment]
Hi,

Please, be specific - what "import statements" do you use, what error do you receive and what is Express?

**Comment by Ristophonics** [comment]
yes sir, okay i used npm to install fancyapps and I can see the files in my project 'node_modules' folders. 

When I use  'import { Fancybox } from "@fancyapps/ui";    in my index.js file fancyapps doesn't work, I think because the node_modules folder is outside my root 'app-files' folder ...

I think I just have to create the correct path but then I still need the fancyapps files in 'node_modules' on the server for deployment..

not sure what the right question is I guess....         Do I need node.js running on my server (and this to be a node project) to use the 'node_modules' architecture?  sorry noob lost in the woods next to a path

**Comment by fancyapps** [comment, marked as answer]
Obviously, to use NPM, which stands for `Node package manager`, you need Node. But, it does not matter how you get JS and CSS files, just make sure they are available on your site. If your files are outside root directory, then you need Webpack or something like that, to bundle your assets so that they are available.

Anyway, you can always use CDN links or simply download JS and CSS files and copy inside your webroot.

**Reply by Ristophonics** [reply]
Thank you.
